src/pages/main.tsx

```diff
@@ -1,7 +1,89 @@
 import React from "react";
+import styled from "styled-components";
 
-const MainPage = () => {
-  return <h1>Main page</h1>;
+import { FaAngleRight } from "react-icons/fa";
+import eating from "../img/charEating.png";
+
+const Wrapper = styled.section`
+  background: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const Container = styled.div `
+  width: 394px;
+  height: 670px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const Button = styled.div`
+  width: 65%;
+  height: 140px;
+  border-radius: 10px;
+  background: #F1F1F1;
```

Comment:
colors 파일에 등록해서 사용해주시면 좋을 것 같습니다. 
```suggestion
  background: #F1F1F1;
```